the-collab-lab/tcl-77-smart-shopping-list
# Issue#55: Style Home Page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description



Designed the HomePage views for Unauthenticated and Authenticated audience referencing the [teams' figma design](https://www.figma.com/design/v0vHRdn5Gq8kNJSepE5xM8/TCL--GrocerEase?node-id=0-1&node-type=canvas&t=whiQgh3967Oy6e9O-0). Adjusted some of the default styling coming in from React Bootstrap elements to ensure they align with our design requirements. 

## Related Issue



Closes #55 

## Acceptance Criteria



- [ ]  Align the design as close to the [teams' figma design](https://www.figma.com/design/v0vHRdn5Gq8kNJSepE5xM8/TCL--GrocerEase?node-id=0-1&node-type=canvas&t=whiQgh3967Oy6e9O-0) as possible
- [ ]  The styles are responsive across mobile and tablet screens. 

## Type of Changes

`design` `UI updates`

## Updates

### Before


<img width="767" alt="Screenshot 2024-10-14 at 3 25 46 PM" src="https://github.com/user-attachments/assets/f3d8d46a-6f94-41da-a968-a9c4203a085a">

<img width="707" alt="Screenshot 2024-10-14 at 3 27 01 PM" src="https://github.com/user-attachments/assets/0fc5dcd8-a2d4-41fd-b4ab-776dc1e668c5">


### After



Unauthenticated HomePage:
<img width="424" alt="Screenshot 2024-10-14 at 3 18 53 PM" src="https://github.com/user-attachments/assets/43d59a15-8911-4899-86d8-faec45655cb9">

<img width="771" alt="Screenshot 2024-10-14 at 3 19 25 PM" src="https://github.com/user-attachments/assets/25a08d0f-52c0-4f59-9c53-d7ea22779a70">

Authenticated HomePage:
<img width="428" alt="Screenshot 2024-10-14 at 3 32 46 PM" src="https://github.com/user-attachments/assets/7838ca41-ad18-4ce4-ac2d-5bfc2617274d">

<img width="732" alt="Screenshot 2024-10-14 at 3 33 04 PM" src="https://github.com/user-attachments/assets/654c2bac-7cb1-4454-8bd9-e6496bc6e92c">



## Testing Steps / QA Criteria

1. Visit the [preview URL](https://tcl-77-smart-shopping-list--pr65-fz-design-homepage-hzadu9e3.web.app/) for this PR. 
2. In your browser dev tools, inspect the styling at 425px screen width for mobile, and 768px for tablet for unauthenticated and authenticated HomePage. 
3. Check the functionality of all buttons and interactive elements to ensure that nothing is broken.
